doc/admin/deployment.md

```diff
@@ -1121,7 +1126,10 @@ nodes before starting it.
 
 The DAOS security framework relies on certificates to authenticate
 administrators. The security infrastructure is currently under
-development and will be delivered in DAOS v1.0. Initial support for certificates has been added to DAOS and can be disable either via the command line or in the DAOS Agent configuration file.
+development and will be delivered in DAOS v1.0. Initial support for certificates
+has been added to DAOS and can be disabled either via the command line or in the
+DAOS Agent configuration file. Currently the easiest way to disable certificate
+support is to pass the -i flag to daos_agent.
```

Comment:
Need to escape the underscore in daos_agent?

Reply:
Good catch. Probably need to do the same for daos_server further up in the document.